lib/contentful.ts

```diff
@@ -8,31 +8,107 @@ import { createHttpClient, getUserAgentHeader } from 'contentful-sdk-core'
 import { ContentfulClientApi } from './create-contentful-api'
 import createGlobalOptions from './create-global-options'
 import { makeClient } from './make-client'
-import type { AxiosRequestConfig } from 'axios'
+import type { AxiosAdapter, AxiosRequestConfig } from 'axios'
 import { validateRemoveUnresolvedParam, validateResolveLinksParam } from './utils/validate-params'
 
 export type ClientLogLevel = 'error' | 'warning' | 'info' | string
 
+/**
+ * Client initialization parameters
+ */
 export interface CreateClientParams {
+  /**
+   * Space ID
+   */
   space: string
+  /**
+   * Contentful CDA Access Token
+   */
   accessToken: string
+  /**
+   * Contentful Environment ID
+   * @defaultValue "master"
+   */
   environment?: string
+  /**
+   * Requests will be made over http instead of the default https
+   * @defaultValue true
```

Comment:
I tried formatting default values as code but they still look different to e.g. parameter types which are also code.